util/exercise/src/cmd/generate.rs

```diff
@@ -246,7 +230,33 @@ pub fn generate_exercise(exercise_name: &str, use_maplit: bool) -> Result<()> {
                 &exercise_name
             );
 
-            test_file.write_all(b"// Add your tests here\n")?;
+            let mut test_file = File::create(test_file_name)?;
+
+            test_file.write_all(
+                &format!(
+                    "//! Tests for {exercise_name} \n\
+                     //! \n\
+                     //! Generated by [utility][utility]\n\
+                     //! \n\
+                     //! [utility]: https://github.com/exercism/rust/tree/master/util/exercise\n\
+                     \n",
+                    exercise_name = exercise_name,
+                )
+                .into_bytes(),
+            )?;
+            if use_maplit {
+                test_file.write_all(b"#[macro_use]\nextern crate maplit;\n")?;
+            }
+
+            test_file.write_all(
+                &format!(
+                    "extern crate {escaped_exercise_name};\n\
```

Comment:
As this track now uses the 2018 edition by default, we can remove the `extern crate` line.

Reply:
I'll remove that.
Following this advice, what about also changing 
```
#[macro_use]
extern crate maplit;
```
to 
`use maplit::*;`